generate.py

```diff
@@ -1,23 +1,36 @@
 import pandas as pd
+import requests
 from jinja2 import Environment, FileSystemLoader
 
+good_reads_api_key = input("Enter Goodreads Api Key: ")
```

Comment:
Could you make this load from a sort of config file? This way I don't need to input this each time the page is generated.

Reply:
Updated to load from `config.ini`.